sensor/kubernetes/sensor/sensor.go

```diff
@@ -98,7 +98,8 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 	auditLogCollectionManager := compliance.NewAuditLogCollectionManager()
 
 	o := orchestrator.New(cfg.k8sClient.Kubernetes())
-	complianceService := compliance.NewService(o, auditLogEventsInput, auditLogCollectionManager)
+	complianceMultiplexer := compliance.NewMultiplexer()
+	complianceService := compliance.NewService(o, auditLogEventsInput, auditLogCollectionManager, complianceMultiplexer.GetCommandsC())
```

Comment:
Add to the followup ticket that auditLogEventsInput and auditLogCollectionManager should turn into ComplianceComponent if sensible